src/zalopay/services/resource.ts

```diff
@@ -1,27 +1,33 @@
 import Service from "../service";
-import {Method} from "axios";
+import { Method } from "axios";
+import { RequestOptions } from "../utils/requestUtils";
 
 abstract class Resource {
-    protected endpoint: string;
-    private service: Service;
+  protected endpoint: string;
+  private service: Service;
 
-    protected constructor(service: Service, endpoint: string) {
-        this.service = service;
-        this.endpoint = endpoint;
-    }
+  protected constructor(service: Service, endpoint: string) {
+    this.service = service;
+    this.endpoint = endpoint;
+  }
 
-    public request(method: Method, data: string): Promise<string> {
-        const headers = {
-            Accept: "application/json",
-            "Content-Type": "application/json",
-        };
-        return this.service.httpClient.request<Method, string>({
-            method,
-            url: this.endpoint,
-            data,
-            headers,
-        });
-    }
+  public request(
+    method: Method,
+    data: string,
+    requestOptions?: RequestOptions
```

Comment:
should put "," at the end of line. Because if we add new field, the code change marker will not mark old field. 
It helps reviewers see changes clearly

Reply:
Because the prettier delete this "," here, so for now I don't think this is necessary.
The root problem is the old version of the code does not follow the prettier standard. So if we apply these formats soon, this issue won't happen again.